test/config.sh

```diff
@@ -60,16 +60,16 @@ upload_executable() {
 remote() {
     rem=$1
     shift 1
-    "$@" > >(while read line; do echo -e "\e[0;34m$rem>\e[0m $line"; done)
+    "$@" > >(while read line; do echo -e $"\e[0;34m$rem>\e[0m $line"; done)
```

Comment:
Why do we want this `$` ?  (I read the man-page and still don't know)
"A double-quoted string preceded by a dollar sign ($) will cause the string to be translated according to  the  current  locale."